pyproject.toml

```diff
@@ -1,3 +1,84 @@
+# ===== Project info
+
+[project]
+dynamic = ["version"]
+name = "fastplotlib"
+description = "Next-gen fast plotting library running on WGPU using the Pygfx rendering engine "
+readme = "README.md"
+license = { file = "LICENSE" }
+authors = [{ name = "Kushal Kolar" }, { name = "Caitlin Lewis" }]
+keywords = [
+    "visualization",
+    "science",
+    "interactive",
+    "pygfx",
+    "webgpu",
+    "wgpu",
+    "vulkan",
+    "gpu",
+]
+requires-python = ">= 3.10"
+dependencies = [
+    "numpy>=1.23.0",
+    "pygfx>=0.9.0",
+    "wgpu>=0.20.0",
+    "cmap>=0.1.3",
+    # (this comment keeps this list multiline in VSCode)
+]
+
+[project.optional-dependencies]
+docs = [
+    "sphinx",
+    "sphinx-gallery",
+    "pydata-sphinx-theme",
+    "glfw",
+    "ipywidgets>=8.0.0,<9",
+    "sphinx-copybutton",
+    "sphinx-design",
+    "pandoc",
+    "imageio[ffmpeg]",
+    "matplotlib",
+    "scikit-learn",
+]
+notebook = [
+    "jupyterlab",
+    "jupyter-rfb>=0.5.1",
+    "ipywidgets>=8.0.0,<9",
+    "sidecar",
+]
+tests = [
+    "pytest",
+    "nbmake",
+    "black",
+    "scipy",
+    "imageio[ffmpeg]",
+    "scikit-learn",
+    "tqdm",
+]
+imgui = ["imgui-bundle"]
+dev = ["fastplotlib[docs,notebook,tests,imgui]"]
+
+[project.urls]
+Homepage = "https://www.fastplotlib.org/"
+Documentation = "https://www.fastplotlib.org/"
+Repository = "https://github.com/fastplotlib/fastplotlib"
+
+# ===== Building
+
 [build-system]
-requires = ["setuptools", "wheel"]
+requires = ["flit_core >=3.2,<4"]
```

Comment:
I just look into flit, it seems nice! Do we need to add an `[include]` section to replace the MANIFEST.in?

Reply:
Nope, Flit by default includes everything under the `fastplotlib` dir. I double-checked that the file in `assets` are in the distribution. The directory `fastplotlib/utils/colormap` does not exist, so I think this line in the manifest is outdated.

Reply:
Ah thanks!